Sources_v3/StreamChatUI/Utils/UIFont+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+extension UIFont {
+    func withTraits(traits: UIFontDescriptor.SymbolicTraits) -> UIFont {
+        let descriptor = fontDescriptor.withSymbolicTraits(traits)
+        return UIFont(descriptor: descriptor!, size: 0)
```

Comment:
The size should probably be `self.size`?

Reply:
I've checked that `size: 0` also intended to preserve the original size.

Reply:
Will try with `self.size` - it will be more clear in that case.